apps/crn-server/test/routes/research-outputs.route.test.ts

```diff
@@ -126,9 +126,7 @@ describe('/research-outputs/ route', () => {
     const researchOutputResponse = getResearchOutputResponse();
 
     test('Should return a 201 when is hit', async () => {
-      const createResearchOutputRequest = {
-        ...getResearchOutputPostRequest(),
-      };
+      const { ...createResearchOutputRequest } = getResearchOutputPostRequest();
```

Comment:
i realise it was like that previously, but the spread syntax is unnecessary here. You can just do 
```suggestion
      const createResearchOutputRequest = getResearchOutputPostRequest();
```
as we dont need a shallow clone